nativescript-angular/directives/dialogs.ts

```diff
@@ -159,7 +177,13 @@ export class ModalDialogService {
 
             // TODO: remove <any> cast after https://github.com/NativeScript/NativeScript/pull/5734
             // is in a published version of tns-core-modules.
-            (<any>parentView).showModal(componentView, context, closeCallback, fullscreen, animated, stretched);
+            if (ios) {
+                // tslint:disable-next-line:max-line-length
+                (<any>parentView).showModal(componentView, { context, closeCallback, fullscreen, animated, stretched, ios });
+            } else {
+                (<any>parentView).showModal(componentView, context, closeCallback, fullscreen, animated, stretched);
+            }
+            this.componentView = componentView;
```

Comment:
Can you remove this method. After locally testing and reviewing this PR I discovered a bug (https://github.com/NativeScript/NativeScript/issues/7050) that when fixed will resolve this issue you are working around with this code so it is not needed.